### ISSUE TITLE
AuthorizeSecurityGroupIngress fails when FromPort is 0

### ISSUE THREAD

**Issue by mattarmstrong** [aws/aws-sdk-go-v2#1203]
Confirm by changing [ ] to [x] below to ensure that it's a bug:
- [x] I've gone though the [API reference](https://docs.aws.amazon.com/sdk-for-go/v2/api/)
- [x] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-go) for answers
- [x] I've searched for [previous similar issues](https://github.com/aws/aws-sdk-go-v2/issues) and didn't find any solution
  
**Describe the bug**
Receive an invalidParameterValue error when using `FromPort: 0` inside `IpPermission` when calling `AuthorizeSecurityGroupIngress`:

```
operation error EC2: AuthorizeSecurityGroupIngress, https response error StatusCode: 400, RequestID: XXX-XXX-XXX-XXX-XXX, api error InvalidParameterValue: Invalid value 'Must specify both from and to ports with TCP/UDP.' for portRange.
```

**Version of AWS SDK for Go?**
github.com/aws/aws-sdk-go-v2 v1.3.0

**Version of Go (`go version`)?**
go version go1.16.2 darwin/amd64

**To Reproduce (observed behavior)**
```
_, err := client.AuthorizeSecurityGroupIngress(context.TODO(),
	&ec2.AuthorizeSecurityGroupIngressInput{
		GroupId:       aws.String("sg-xxx"),
		IpPermissions: []types.IpPermission{
			types.IpPermission{
				FromPort:   0,
				ToPort:     65535,
				IpProtocol: aws.String("tcp"),
				IpRanges: []types.IpRange{
					types.IpRange{
						CidrIp:      aws.String("0.0.0.0/0"),
						Description: aws.String(""),
					},
				},
			},
		},
	},
)
```

**Expected behavior**
Expected for operation to be successful

**Comment by lnregalias** [comment]
FYI probably fixed by https://github.com/aws/aws-sdk-go-v2/pull/1195

**Comment by skmcgrail** [comment]
As indicated this issue was resolved in #1195, apologies for not updating this ticket sooner.
